Dockerfile.backend.build

```diff
@@ -0,0 +1,11 @@
+FROM public.ecr.aws/docker/library/node:16
+RUN mkdir -p /home/node/app
+WORKDIR /home/node/app
+COPY . .
```

Comment:
It will be marked as potential security issues in some static scan. Let's explicitly copy the folders/files to the container to avoid copyng sensitive data if they are not added to docker ignore.